Modern/ops/src/main/java/org/bonej/ops/mil/MILPlane.java

```diff
@@ -0,0 +1,294 @@
+
+package org.bonej.ops.mil;
+
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Random;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import net.imagej.ops.Contingent;
+import net.imagej.ops.Op;
+import net.imagej.ops.special.function.AbstractBinaryFunctionOp;
+import net.imagej.ops.special.function.BinaryFunctionOp;
+import net.imagej.ops.special.function.Functions;
+import net.imagej.ops.special.hybrid.BinaryHybridCFI1;
+import net.imagej.ops.special.hybrid.Hybrids;
+import net.imglib2.Interval;
+import net.imglib2.RandomAccess;
+import net.imglib2.RandomAccessibleInterval;
+import net.imglib2.type.BooleanType;
+import net.imglib2.type.numeric.integer.LongType;
+import net.imglib2.type.numeric.real.DoubleType;
+import net.imglib2.util.ValuePair;
+
+import org.bonej.ops.BoxIntersect;
+import org.bonej.ops.RotateAboutAxis;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+import org.scijava.vecmath.AxisAngle4d;
+import org.scijava.vecmath.Point3d;
+import org.scijava.vecmath.Tuple3d;
+import org.scijava.vecmath.Vector3d;
+
+/**
+ * An op that finds the mean intercept length (MIL) vector of an interval.
+ * <p>
+ * A MIL vector is defined as <em>MIL(<b>v</b>) = C(<b>v</b>) / h</em>, where
+ * <em><b>v</b></em> is a set of parallel lines, <em>C(<b>v</b>)</em> is the
+ * number of times <em><b>v</b></em> intercept foreground, and <em>h</em> is the
+ * sum of lengths of <em><b>v</b></em>. The lines <em><b>v</b></em> are traced
+ * through the interval. A line intercepts foreground, when the previous point
+ * sampled along it is background, and the current one is foreground.
+ * </p>
+ * <p>
+ * For example MIL vectors can be used to estimate the anisotropy of the
+ * "texture" in an image. It's best suited for images that are completely
+ * filled, and a part of a larger whole, e.g. a section of trabecular bone.
+ * </p>
```

Comment:
detail: I may be wrong, but "section" implies 2-dimensionality to the physiologist in me... "volume" may be better?